brave/src/main/java/com/linecorp/armeria/server/brave/ArmeriaHttpServerAdapter.java

```diff
@@ -88,6 +89,16 @@ public int statusCodeAsInt(RequestLog requestLog) {
         return requestLog.status().code();
     }
 
+    @Override
+    public String methodFromResponse(RequestLog requestLog) {
+        return requestLog.method().name();
+    }
+
+    @Override
+    public String route(RequestLog response) {
+        return ((ServiceRequestContext) response.context()).route().meterTag();
```

Comment:
- How about guarding it with `instanceof` and returning `null` when the context is not a `ServiceRequestContext`?
- How about adding some TODO like this:
  ```java
  // TODO(kojilin): Do not include `methods` in the returned string.
  ```

Reply:
this is implemented in server adapter, will it be non ServiceRequestContext?

Reply:
Do you think it's possible to use `Route#paths` and `Route#pathType` to make `route`?

e.g.
```
RoutePathType#EXACT || RoutePathType#PREFIX || PARAMETERIZED -> paths.get(1)
REGEX -> paths.get(0),
REGEX_WITH_PREFIX -> paths.get(0) + paths.get(1)

```

Reply:
I think you can use it. Good idea. And assertion should be enough for the context type check. :+1: